src/components/calcite-flow-item/calcite-flow-item.tsx

```diff
@@ -61,6 +61,11 @@ export class CalciteFlowItem {
    */
   @Prop() showBackButton = false;
 
+  /**
+   * Summary text. A description displayed underneath the heading.
+   */
+  @Prop() summary: string;
```

Comment:
Can we put this as optional? `summary?:`?

Reply:
Done.